fiftyone/operators/types.py

```diff
@@ -723,7 +757,11 @@ class Choices(View):
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
-        self.choices = kwargs.get("choices", [])
```

Comment:
I wonder if this is a breaking change

Reply:
I tried to make this entirely backwards compatible - a choice is only included now if it has a `include = True` - which is the default.